### PR TITLE
Unified build script

### PR DESCRIPTION
**Pull request by SinTh0r4s**
One issue: buildcraft 7.1.24 is not available on maven (only 7.1.23), but the newest version is required. I added it as jar, but this will cause transitive issues. Cannot do anything about that for now.

### CODE REVIEW

**Comment by bombcar** [comment]
> One issue: buildcraft 7.1.24 is not available on maven (only 7.1.23), but the newest version is required. I added it as jar, but this will cause transitive issues. Cannot do anything about that for now.

Won’t that fix with https://github.com/GTNewHorizons/BuildCraft migrating?

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+autoUpdateBuildScript = false
+
+minecraftVersion = 1.7.10
+forgeVersion = 10.13.4.1614
+
+# Select a username for testing your mod with breakpoints. You may leave this empty for a random user name each time you
+# restart Minecraft in development. Choose this dependent on your mod:
+# Do you need consistent player progressing (for example Thaumcraft)? -> Select a name
+# Do you need to test how your custom blocks interacts with a player that is not the owner? -> leave name empty
+developmentEnvironmentUserName = "Developer"
```
Comment (bombcar) [inline gradle.properties]: This should have quotes removed

**Comment by SinTh0r4s** [comment]
Ready for final review and merge

**Comment by Dream-Master** [comment]
If the build went trough I will merge it.

**Review by Dream-Master — APPROVED** [review]
